modules/rosetta/src/test/scala/org/tessellation/rosetta/server/RosettaConstructionSuite.scala

```diff
@@ -0,0 +1,151 @@
+package org.tessellation.rosetta.server
+
+import java.util.concurrent.TimeUnit
+
+import cats.effect.unsafe.implicits.global
+import cats.effect.{Async, IO, Resource}
+
+import scala.concurrent.duration.Duration
+import scala.io.Source
+
+import org.tessellation.ext.cats.effect.ResourceIO
+import org.tessellation.ext.crypto._
+import org.tessellation.kryo.KryoSerializer
+import org.tessellation.rosetta.server.JsonObjects.RequestJson
+import org.tessellation.rosetta.server.JsonObjects.implicits._
+import org.tessellation.rosetta.server.runner.runnerKryoRegistrar
+import org.tessellation.sdk.config.types.{HttpClientConfig, HttpServerConfig}
+import org.tessellation.sdk.resources.MkHttpServer.ServerName
+import org.tessellation.sdk.resources.{MkHttpClient, MkHttpServer}
+import org.tessellation.security.SecurityProvider
+import org.tessellation.shared.sharedKryoRegistrar
+
+import com.comcast.ip4s.{Host, Port}
+import io.circe.parser.{decode, parse}
+import io.circe.syntax._
+import io.circe.{Json, JsonNumber}
+import org.http4s._
+import org.http4s.circe._
+import org.http4s.client.Client
+import org.http4s.server.Server
+import org.typelevel.log4cats.SelfAwareStructuredLogger
+import org.typelevel.log4cats.slf4j.Slf4jLogger
+import suite.ResourceSuite
+import weaver.scalacheck.Checkers
+
+object RosettaConstructionSuite extends ResourceSuite with Checkers {
+  override type Res = (Resource[IO, Server], Resource[IO, Client[IO]])
+  override def sharedResource: Resource[IO, Res] = {
+    val server = serverResource()
+    val client = MkHttpClient
+      .forAsync[IO]
+      .newEmber(HttpClientConfig(Duration(5, TimeUnit.SECONDS), Duration(10, TimeUnit.SECONDS)))
+
+    IO.pure((server: Resource[IO, Server], client: Resource[IO, Client[IO]])).asResource
+  }
+
+  test("Construction test") { implicit resources: Res =>
+    val serverObj = resources._1
+    val httpClientObj = resources._2
+
+    serverObj.use { server =>
```

Comment:
_For anyone that can answer this_: Why do I have to have `server =>` here for the tests to run even though I don't call it ever?

Reply:
This can just be `_ =>` -- it has to be invoked here to start the server in background but not actually referenced.